lib/Mojolicious/Sessions.pm

```diff
@@ -45,14 +45,18 @@ sub store {
   # Generate "expires" value from "expiration" if necessary
   my $expiration = $session->{expiration} // $self->default_expiration;
   my $default    = delete $session->{expires};
-  $session->{expires} = $default || time + $expiration
+  
+  my $time = time;
+  $session->{expires} = $default || $time + $expiration
     if $expiration || $default;
+  my $max_age = $session->{expires} - $time if $session->{expires};
```

Comment:
Don't do `my $foo = ... if ...`.

Reply:
Okay. What should I do instead? A traditional if?

my $max_age;
if ($session->{expires}) {
    $max_age = $time if $session->{expires};
}

Reply:
Try to find the shortest solution that's still idiomatic Perl.